skeleton-es2016-webpack/webpack.prod.config.js

```diff
@@ -35,11 +35,11 @@ module.exports = {
     loaders: [
       { test: /\.js$/, loader: 'babel', exclude: /node_modules/, query: {stage: 0} },
       { test: /\.css?$/, loader: 'style!css' },
-      { test: /\.html$/, loader: 'raw' },
-      { test: /\.(png|gif|jpg)$/, loader: 'url-loader?limit=8192' },
-      { test: /\.woff2(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url-loader?limit=10000&minetype=application/font-woff2' },
-      { test: /\.woff(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url-loader?limit=10000&minetype=application/font-woff' },
-      { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'file-loader' }
+      { test: /\.html$/, loader: 'html' },
+      { test: /\.(png|gif|jpg)$/, loader: 'url?limit=8192' },
+      { test: /\.woff2(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url?limit=10000&minetype=application/font-woff2' },
+      { test: /\.woff(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url?limit=10000&minetype=application/font-woff' },
+      { test: /\.(ttf|eot|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'file' }
```

Comment:
@martijnboland, why don't You reuse conf from one file for another? In addition to module.loaders (that is 100% the same), some more bits of conf could be reused or extended (I don't have strong opinion whether common configuration should be in separate file or not - it depends how much real-life application conf differs from what you have here).